test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -763,6 +764,21 @@ public Settings indexSettings() {
         return builder.build();
     }
 
+    /**
+     * Setting all feature flag settings at base IT, which can be overridden later by individual
+     * IT classes.
+     *
+     * @return Feature flag settings.
+     */
+    protected Settings featureFlagSettings() {
+        return Settings.builder()
```

Comment:
I don't think we need to set these defaults here, all settings should be registered as false by default in SettingsModule.

Also on that note, I think we should add a test case that validates all `BUILT_IN_FEATURE_FLAGS` are by default false.

Reply:
+1